src/mscorlib/src/System/Threading/Tasks/TaskCache.cs

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.Threading.Tasks
+{
+    internal static class TaskCache
```

Comment:
There's already a task cache in mscorlib, e.g.
https://github.com/dotnet/coreclr/blob/master/src/mscorlib/src/System/Runtime/CompilerServices/AsyncMethodBuilder.cs#L801
We shouldn't be adding a separate one.